Incorrect comparision operator that doesn't work in strict POSIX shells.

--- configure.orig	2009-04-27 00:07:12.146716716 +0200
+++ configure	2009-04-27 00:07:31.506853126 +0200
@@ -11255,7 +11255,7 @@ $as_echo "$as_me: error: unsupported ope
    { (exit 1); exit 1; }; }
 esac
 
- if test "x$backend" == "xlinux"; then
+ if test "x$backend" = "xlinux"; then
   OS_LINUX_TRUE=
   OS_LINUX_FALSE='#'
 else
